cmd/crc/cmd/start.go

```diff
@@ -250,3 +256,25 @@ func commandLinePrefix(shell string) string {
 	}
 	return "$"
 }
+
+func checkDaemonStarted() error {
+	if config.Get(cmdConfig.NetworkMode).AsString() == string(network.DefaultMode) {
+		return nil
+	}
+	daemonClient := daemonclient.New()
+	version, err := daemonClient.APIClient.Version()
+	if err != nil {
+		return fmt.Errorf(daemonStartedErrorMessage(), err)
```

Comment:
This could start the daemon rather than returning an error, at least in the !installer case.